content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,74 @@
+from this branch would be considered highly experimental. While the branch builds locally, it fails the CI pipeline when
+another PR is added. With that said, there are still people and companies that are using artifacts from this branch
+in both staging and production environments.
+
+The original design was created with customer requirements that specified client originating IPv4 requests should be proxied over IPv4, and the same for an originating IPv6 requests to be satisfied over IPv6 (we will refer to this as native IP family going forward). To do this, we initially had to duplicate Envoy configuration for listeners, clusters, routes and endpoints. Given that many people already experience Envoy memory and CPU consumption issues, early feedback wanted us to completely re-evaluate this approach. Many proxies transparently handle outbound dual-stack traffic regardless of how the traffic was originated. Much of the earliest feedback was to implement the same behavior in Istio and Envoy.
+
+## Redefining Dual Stack Support
+
+Much of the feedback provided by the community for the original RFC was to update Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them to fit a simplified design.
+
+## Support for Dual Stack in Istio 1.17
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have implemented [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolved, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes made by the team can be found at their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on at least a half dozen more to make Envoy adoption of
```

Comment:
For anyone wanting to try the feature this is a little scary.  Are these big things or little things?  It is just to make things easier or are some use cases broken? If there is a running list somewhere of what is in progress adding a link might help.   If not just changing the wording to make this less scary may help.

Reply:
It's preferable for people to use the 1.17 branch. The experimental branch is out of date. There may still continue to be things to learn from it, however.